am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -36,8 +36,9 @@
         + "and access_management.service_name = :resourceDefinition.serviceName "
         + "and access_management.resource_type = :resourceDefinition.resourceType "
         + "and access_management.resource_name = :resourceDefinition.resourceName "
-        + "and access_management.attribute = :attributeAsString "
-        + "or access_management.attribute like concat(:attributeAsString, '/', '%')")
+        + "and (:relationship is null or access_management.relationship = :relationship) "
+        + "and (access_management.attribute = :attributeAsString "
```

Comment:
I'd suggest having `or` on the same line for better readability.

Reply:
👍